tests/wpt/web-platform-tests/webmessaging/Channel_postMessage_with_transfer_entangled.htm

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title> When transferring both a port
+    and the one it is entangled with,
+    pre-transfer enqueued messages should be received. </title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id=log></div>
+<script>
+async_test(function(t) {
+    var channel1 = new MessageChannel();
+    var channel2 = new MessageChannel();
+
+    // One, send a message.
+    channel1.port1.postMessage(1);
+
+     // Two, transfer both ports.
+     channel2.port1.postMessage("transfer", [channel1.port1]);
```

Comment:
Note: I was doing both transfers in one call to `postMessage("transfer", [channel1.port1, channel1.port2]);`, and the test was failing intermittently. It turns out that the transferred port are not always received in the same order as the original array. So `evt.ports[0]` does not necessarily have to point to the first port in the array used in the `postMessage` call. 

Could be a bug somewhere on our side, but I think one just can't rely on ordering of the `transfer` argument in the context of the structured clone API.